dev/com.ibm.ws.security.jwtsso/test/com/ibm/ws/security/jwtsso/JwtSsoTest.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+ package com.ibm.ws.security.jwtsso;
+ 
+ public class JwtSsoTest{
+    // todo: unit tests here.
```

Comment:
This might actually cause a failure in a build since there are no `@Test` methods.

Reply:
it works ok in sandbox, let's find out!